src/main/java/com/depromeet/domain/member/dao/MemberRepositoryImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.depromeet.domain.member.dao;
+
+import static com.depromeet.domain.member.domain.QMember.*;
+import static com.depromeet.domain.mission.domain.QMission.*;
+import static com.depromeet.domain.missionRecord.domain.QMissionRecord.*;
+
+import com.depromeet.domain.member.domain.Member;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class MemberRepositoryImpl implements MemberRepositoryCustom {
+
+    private final JPAQueryFactory jpaQueryFactory;
+
+    @Override
+    public List<Member> findNonCompleteMissions(LocalDateTime now) {
+        return jpaQueryFactory
+                .selectFrom(member)
+                .leftJoin(member.missions, mission)
+                .fetchJoin()
+                .leftJoin(mission.missionRecords, missionRecord)
+                .on(missionRecord.id.eq(mission.id))
```

Comment:
```suggestion
                .on(missionRecord.mission.id.eq(mission.id))
```

요기 missionRecord.mission.id로 가야하지 않을까용?

Reply:
넵 맞습니다 그런데 방금 쿼리 수정해보면서
```java
.selectFrom(member)
                .leftJoin(member.missions, mission)
                .fetchJoin()
                .leftJoin(mission.missionRecords, missionRecord)
                .on(missionRecord.createdAt.between(start, end))
                .where(
```

위 on 절만 걸어줘도 아래 sql 처럼 동작되는 것을 확인했어용 혹시 그래도 id 검증 쿼리가 필요하다면 넣도록 하겠습니다!
```sql
from
        member m1_0 
    left join
        mission m2_0 
            on m1_0.member_id=m2_0.member_id 
    left join
        mission_record m3_0 
            on m2_0.mission_id=m3_0.mission_id 
            and m3_0.created_at between ? and ? 
    where
```